### PR TITLE
Add create new robot variants file

### PR DESCRIPTION
**Pull request by muscatd75**

### CODE REVIEW

File: codeGenerator/configuration/configuration.xml [hunk 1]
```diff
@@ -1,32 +1,21 @@
...
-<GenerationNotice>
-//==============================================================
-// This file is auto generated by FRCrobotCodeGen302.exe Version $CODE_GENERATOR_VERSION$ 
-// Changes to this file may cause incorrect behavior and will be lost when 
-// the code is regenerated, unless the changes are delimited by:
-//==============================================================
-</GenerationNotice>
-
-<CopyrightNotice>
+  <rootOutputFolder>C:\FRC\2023SummerProject\src\main\cpp</rootOutputFolder>
```
Comment (nickfrisch) [inline codeGenerator/configuration/configuration.xml:3]: Why is this an absolute path to a local directory?

Comment (muscatd75) [reply codeGenerator/configuration/configuration.xml:3]: not sure, I need to check

File: codeGenerator/configuration/configuration.xml [hunk 2]
```diff
@@ -1,32 +1,21 @@
...
+  <robotConfiguration>C:\FRC\2023SummerProject\codeGenerator\configuration\2023SummerRobots.xml</robotConfiguration>
```
[the remark this answers is not in the document]
Comment (muscatd75) [reply codeGenerator/configuration/configuration.xml:4]: not sure, I need to check

**Comment by muscatd75** [comment]
This pull request is irrelevant

**Pull request closed without merging**